### ISSUE TITLE
Additional error measures

### ISSUE THREAD

**Issue by ercbk** [RamiKrispin/TSstudio#44]
RMSE and MAPE have their issues in certain contexts, so I just like to put in a request for additional error measures such as MASE, MAE, Huber, etc., or maybe a function or argument that allows the user to add their own.

Beautiful visuals. Thank you for package.

**Comment by RamiKrispin** [comment]
Hi @ercbk, thanks for the feedback. I will add more error metrics for the next version

**Comment by jam1245** [comment]
Hi! I'd love to take a crack at implementing this......

After reviewing the `train_model()` source, my proposed approach would be to:

1. **Add MAE and MASE to the leaderboard output** alongside the existing `avg_mape` and `avg_rmse` columns. MASE would be computed as:

```r
MASE <- mean(abs(actual - forecast)) / mean(abs(diff(train)))
```

...where the denominator is the mean absolute difference of the training series (the naive in-sample benchmark), computed once per backtesting partition and averaged across partitions in the leaderboard.

2. **Extend the `error` argument** to accept `"MAE"` and `"MASE"` as valid sort criteria in addition to the existing `"MAPE"` and `"RMSE"`, so users can rank models by whichever metric fits their use case.

3. Optionally, expose a **`custom_error`** function argument for power users who want to supply their own loss function — though I'd suggest keeping that as a stretch goal if the maintainer prefers to keep scope narrow.

A question.....
- Should MASE use the **training partition naive benchmark** (mean absolute first difference of the training set) or a **seasonal naive benchmark** (lag-h difference)? The seasonal naive version is more defensible for seasonal series like `USgas`, but adds a `frequency` dependency.
